packages/gamut/src/AppWrapper/index.tsx

```diff
@@ -1,15 +1,23 @@
+import { shouldForwardProp } from '@codecademy/gamut-styles';
+import { Theme } from '@emotion/react';
 import styled from '@emotion/styled';
 
+export interface AppWrapperProps {
+  backgroundColor?: keyof Theme['colors'];
+}
+
 /**
  *
  * This element safely resets the stacking context with limited side effects.
  * https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context
  *
  * **WARNING**: Please do not change the values of position or z-index here or extend this component with overrides
- * to these propertiesas it will no longer guarantee a predictable behavior
+ * to these properties as it will no longer guarantee a predictable behavior
  */
 
-export const AppWrapper = styled.div`
+export const AppWrapper = styled('div', { shouldForwardProp })<AppWrapperProps>`
+  background-color: ${({ backgroundColor = 'white', theme }) =>
+    theme.colors[backgroundColor]};
   position: relative;
   z-index: 1;
 `;
```

Comment:
To ensure color modes work as expected we should maybe switch this into 2 components, one to reset the context for all portals / mounting nodes and then extend it with the Background component for our main App tree :).

```suggestion
/** Use this in BodyPortal */
export const RootWrapper = styled.div`
  position: relative;
  z-index: 1;
`;

/** Use this for Pages */
const AppWrapper = RootWrapper.withComponent(Background);
```

Reply:
@codecaaron I was thinking about this, but: we _only_ want `background-color` to be configurable. Is there a thing for that?

Reply:
Not currently, ostensibly we could artificially limit the types of Background so only `bg` is configurable here. 

Is the worry here that we could end up adding the wrong props to Background globally and cause issues?

Reply:
Yeah, I'd like to limit the API if possible. What do you think? Is the effort worth it?

Reply:
Let's try this then:

```tsx
const PageContainer = forwardRef<HTMLDivElement, Pick<ComponentProps<typeof Background>, 'bg'>>((props, ref)) => (<Background {...props}  ref={ref} />));

const AppWrapper = RootWrapper.withComponent(Background);
```

Reply:
Talked on Slack: I'll do this as a fast follow; wanna ship this now to unblock portal-app stuff.

Reply:
https://codecademy.atlassian.net/browse/WEB-1455